lib/features/auth/presentation/login/login_page.dart

```diff
@@ -1,275 +1,108 @@
-                        svgIcon: "assets/images/password-type.svg",
-                        type: FixitTextType.password,
-                      ),
-                      Row(
-                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                        children: [
-                          Expanded(
-                            child: Container(
-                              margin: const EdgeInsets.only(
-                                  right: 10, left: 10, top: 10),
-                              padding: const EdgeInsets.all(10),
-                              child: Row(
-                                crossAxisAlignment: CrossAxisAlignment.start,
-                                children: <Widget>[
-                                  const SizedBox(
-                                    width: 10,
-                                  ),
-                                  SvgPicture.asset(
-                                      "assets/images/arrow-circle-left.svg"),
-                                  const SizedBox(
-                                    width: 10,
-                                  ),
-                                  Expanded(
-                                    child: InkWell(
-                                      onTap: showForgetPasswordBottomSheet,
-                                      child: Text(
-                                        Strings.forgetPasswordQuestion,
-                                        overflow: TextOverflow.clip,
-                                        style: const TextStyle(
-                                          fontSize: 15,
-                                          color: Constant.violetColorDark,
-                                          fontFamily: AppFontFamily.regular,
-                                        ),
-                                      ),
-                                    ),
-                                  )
-                                ],
-                              ),
-                            ),
-                          ),
-                        ],
-                      ),
-                      Container(
-                        margin:
-                            const EdgeInsets.only(left: 16, right: 16, top: 32),
-                        child: SafeArea(
-                          child: FixitButton(
-                            enable: sendButtonEnable,
-                            loading: state is LoginLoading,
-                            label: Strings.login,
-                            background: Constant.violetColorDark,
-                            labelColor: Colors.white,
-                            onTab: onLogin,
+    return BlocProvider(
+      create: (context) => sl<LoginFormBloc>(),
+      child: Builder(
+        builder: (context) {
+          final formBloc = BlocProvider.of<LoginFormBloc>(context);
+          return Theme(
+              data: Theme.of(context).copyWith(
```

Comment:
### Great work 👍

But you can move this theme to the global theme in **MaterialApp**, to use it in all your project